daemon/main.go

```diff
@@ -307,14 +303,14 @@ func checkMinRequirements() {
 func init() {
 	cobra.OnInitialize(initConfig)
 	flags := RootCmd.Flags()
-	flags.StringVar(&config.AccessLog,
+	flags.StringVar(&option.GetConfig().AccessLog,
```

Comment:
As a nit, this looked a little weird to me repeated over and over. If we assigned `config` local to the function it would also work. It might just be all the lines that needed changing, or that `GetConfig` sounds wrong to me (I can't think of a better name, though).